src/geometryRTheta/advection_field/advection_field_rtheta.hpp

```diff
@@ -484,7 +495,10 @@ class AdvectionFieldFinder
             CoordRTheta const coord_rtheta(ddc::coordinate(irtheta));
 
             DTensor<VectorIndexSet<R_cov, Theta_cov>, VectorIndexSet<R_cov, Theta_cov>> inv_G
-                    = metric_tensor.inverse(coord_rtheta);
+                    = metric_tensor.inverse(coord_rtheta); // inverse tensor metric
```

Comment:
Similar question to above. Is it really that hard to recognise that `metric_tensor.inverse` is the inverse tensor metric (or rather the inverse metric tensor)?